packages/graphql-client/README.md

```diff
@@ -36,8 +38,9 @@ const client = createGraphQLClient({
 | Name       | Type                  | Description                                                      |
 | ---------- | --------------------- | ---------------------------------------------------------------- |
 | variables? | `OperationVariables` | Variable values needed in the graphQL operation                  |
-| url?       | `string`              | Althernative request API URL                                     |
+| url?       | `string`              | Alternative request API URL                                     |
 | headers?   | `{[key: string]: string}`             | Additional and/or replacement headers to be used in the request |
+| retries?   | `number`             | Alternative number of retries for the request. Retries only occur for requests that were abandoned or if the server responds with a `Too Many Request (429)` or `Service Unavailable (503)` response. |
```

Comment:
Yeah - the last thing we want to do is to flood the server when there's an actual outage - let's stick with only retrying for `503s` for now (since this is a status code that does support retries since it's 1 of 3 status' that can return a `retry-after` header) and then we can see if devs gives other feedback and we can reevaluate.